config/config.go

```diff
@@ -133,3 +141,18 @@ type (
 		MigratorParallelSlabsPerWorker uint64        `yaml:"migratorParallelSlabsPerWorker,omitempty"`
 	}
 )
+
+var DefaultSQLiteConfig = SQLite{
+	Database:        "db.sqlite",
+	MetricsDatabase: "metrics.sqlite",
```

Comment:
What's the benefit of making this configurable? From what I can tell the actual location of the sqlite files can't be changed to outside the `renterd/db` directory meaning this is only renaming the file?

Reply:
Hm I don't remember. I think it's probably best to not make it configurable if there is no reason to ever do so.
Making the MySQL db only configurable via config and env var seems alright as well. We don't allow configuring passwords via CLI anyway so it would be a bit weird.
Renaming and deprecating existing config is fine but probably not urgent enough to include it in the same PR as the fix for the issue.

Reply:
Ok. Updated.